MAUI/Barcode/getting-started.md

```diff
@@ -0,0 +1,167 @@
+platform: maui
+control: SfBarcodeGenerator
+documentation: ug
+---
+
+# Getting Started with .NET MAUI Barcode Generator
+
+This section explains the steps required to add the [`.NET MAUI Barcode Generator`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.SfBarcodeGenerator.html) control. This section explains the steps required to add the barcode and set its symbology. This section covers only basic features needed to get started with Syncfusion barcode generator control.
+
+## Adding a SfBarcodeGenerator reference
+
+The Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add SfBarcodeGenerator to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Barcode] and then install it.
+
+### Register the handler
+
+Syncfusion.Maui.Core nuget is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file register the handler for Syncfusion core.
+
+{% highlight C# hl_lines="17" %}
+
+using Microsoft.Maui;
+using Microsoft.Maui.Hosting;
+using Microsoft.Maui.Controls.Compatibility;
+using Microsoft.Maui.Controls.Hosting;
+using Microsoft.Maui.Controls.Xaml;
+using Syncfusion.Maui.Core.Hosting;
+
+namespace GaugeMauiSample
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureSyncfusionCore()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+            });
+
+            return builder.Build();
+        }
+    }
+}
+
+{% endhighlight %}
+
+## Adding a namespace
+
+Add the following namespace to add .NET MAUI Badge Notifications.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    xmlns:barcode="clr-namespace:Syncfusion.Maui.Barcode;assembly=Syncfusion.Maui.Barcode"
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+    using Syncfusion.Maui.Barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Initialize the SfBarcodeGenerator control
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<barcode:SfBarcodeGenerator Value="http://www.syncfusion.com" 
+                            HeightRequest="150"/>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+barcode.HeightRequest = 150;
+barcode.Value = "http://www.syncfusion.com";
+this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Intialize Barcode](images/getting-started/maui-intialize-barcode.png)
+
+N> The default symbology of SfBarcodeGenerator is `Code128`.
+
+## Initialize QR Code symbology
+
+You can set the required symbology type to the barcode generator based on input value by initializing the `Symbology` property. In the following code snippet, the QR code is set as the barcode symbology.
```

Comment:
In the following code sample, the QR code is set as the barcode symbology.

Reply:
Changes updated.